packages/client/src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Bringing in TypeScript _means_ a lot AFAIK you don't _need_ TypeScript to use storybook. Does bringing in TypeScript make sense right now?

Reply:
The typescript dependency if I understood it right, was added during the linting ticket, they said it solve some error they were having. I can look into it further and see if we can't do without it and let you know.